<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>labin.xu</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_new_task">
          <attribute name="title">
           <string>New Task</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>站点：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="lb_current_site">
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>23</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <underline>true</underline>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>ali</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lb_task_name">
               <property name="maximumSize">
                <size>
                 <width>48</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>任务名称:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_task_name">
               <property name="maximumSize">
                <size>
                 <width>316</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="lb_search_conditions">
               <property name="maximumSize">
                <size>
                 <width>66</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>搜索关键字：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_search_keywords">
               <property name="maximumSize">
                <size>
                 <width>420</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="pb_Ok">
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_Cancel">
               <property name="text">
                <string>清空</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_running_tasks">
          <attribute name="title">
           <string>Running Tasks</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>Tasks</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="TaskRunningTable" name="lw_processing_tasks">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <column>
                 <property name="text">
                  <string>任务</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>进度</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Task Details</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableWidget" name="tw_processing_task_details"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_completed_tasks">
          <attribute name="title">
           <string>Completed</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="maximumSize">
              <size>
               <width>191</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>Tasks</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="ListWidget" name="lw_finished_tasks">
                <property name="maximumSize">
                 <size>
                  <width>171</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Task Details</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QTableWidget" name="tw_finished_task_details"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_history">
          <attribute name="title">
           <string>History</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_search">
          <attribute name="title">
           <string>Search</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="actionDeleteFromDisk"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionDebug"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>调试</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionDeleteFromDisk">
   <property name="text">
    <string>从磁盘删除</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaskRunningTable</class>
   <extends>QTableWidget</extends>
   <header>tablewidgetwithprocess.h</header>
   <slots>
    <signal>cellContextClicked(QTableWidgetItem*)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>listwidget.h</header>
   <slots>
    <signal>cellContextClicked(QListWidgetItem*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lw_processing_tasks</tabstop>
  <tabstop>tw_processing_task_details</tabstop>
  <tabstop>lw_finished_tasks</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tw_finished_task_details</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lw_finished_tasks</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onFinishedTasksItemClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Ok</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onNewTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lw_processing_tasks</sender>
   <signal>cellContextClicked(QTableWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onProcessTasksRClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lw_finished_tasks</sender>
   <signal>cellContextClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onFinishedTasksRClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabBarClicked(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTabBarClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lb_current_site</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectSite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>onFinishedTasksItemClicked()</slot>
  <slot>onNewTask()</slot>
  <slot>onProcessTasksRClicked()</slot>
  <slot>onFinishedTasksRClicked()</slot>
  <slot>onTabBarClicked()</slot>
  <slot>onSelectSite()</slot>
 </slots>
</ui>
